examples/fullstack/backend/README.md

```diff
@@ -77,7 +77,8 @@ Listening on :8080
 From another terminal, send a JSON request:
 
 ```shell
-$ curl -X POST -H "content-type: application/json" -d '{ "name": "World" }' -i -w "\n" localhost:8080
+$ curl -X POST -H "content-type: application/json" \
```

Comment:
Can we remove the "$" so this snippet is copy-pastable?

Reply:
I can, but what we really need to do is update the templates that generate these projects, which are based on our other examples.... Should probably open an issue for that. But I'll fix this for now.